docs/user/troubleshooting.md

```diff
@@ -99,6 +99,237 @@ This is safe to ignore and merely indicates that the etcd bootstrapping is still
 
 The easiest way to get more debugging information from the installer is to check the log file (`.openshift_install.log`) in the install directory. Regardless of the logging level specified, the installer will write its logs in case they need to be inspected retroactively.
 
+### Installer fails to initialize the cluster
```

Comment:
nit: I personally prefer Sentence case, but the convention in this file seems to be Title Case (at least for level-three headers).

```console
$ git describe
unreleased-master-459-g2b65c9d4f
$ git grep -h '^### ' docs/user/troubleshooting.md
### No Worker Nodes Created
### Kubernetes API is Unavailable
### Unable to SSH into Master Nodes
### Troubleshooting the Bootstrap Node
### etcd Is Not Running
### Installer Fails to Create Resources
### Check for Pending or Crashing Pods
### One or more nodes are never Ready (Network / CNI issues)
```

Dunno about that last one (Great valley Case? ;).  Can we update this to `### Installer Fails to Initialize the Cluster` or update the rest of the doc to use sentence case for h2 and below?